crates/re_types/src/components/radius_ext.rs

```diff
@@ -4,19 +4,54 @@ impl Radius {
     /// Zero radius.
     pub const ZERO: Self = Self(0.0);
 
-    /// Unit radius.
+    /// Radius of length 1 in scene units.
     pub const ONE: Self = Self(1.0);
```

Comment:
```suggestion
    pub const ONE_IN_SCENE_UNITS: Self = Self(1.0);
```

…but I wonder what the use of this even is?

I guess if we call it `SCENE_UNITS` then users could write `2.0 * Radius::SCENE_UNITS` if we overload `Mul`

Reply:
as before with re_renderer::SizeL: bit of of a mouthful, let's just remove it :shrug: